assets/styles/analysis.less

```diff
@@ -1,41 +1,56 @@
-#analysis {
-	.chart-holder {
-		padding-left: 8px;
-		padding-right: 8px;
-		height: 150px;
-		width: 250px;
-	}
-	.feature-column {
-		width: 266px;
-	}
-	.horizontal-padding {
-		padding-left: 8px;
-		padding-right: 8px;
-	}
-  selection-slider {
-    > * {
-      // Diagram margin left 50px minus 4px slider bubble offset
-      margin-left: 46px;
-      // Diagram margin right 20px minus 4px slider bubble offset
-      margin-right: 16px;
-    }
+.md-padding{
+  max-width: 80%;
 
-    .rzslider {
-      margin-top: 15px;
-    }
+  #analysis {
 
-    input.sliderbox {
-      width: 79px;
+    md-content {
+      overflow-x: auto;
     }
-  }
-  selection-checkboxes {
-    md-checkbox.sliceCheckbox {
-      padding-right: 8px;
+
+  	.chart-holder {
+  		padding-left: 8px;
+  		padding-right: 8px;
+  		height: 150px;
+  		width: 250px;
+  	}
+
+  	.feature-column {
+  		width: 266px;
+      overflow: initial;
```

Comment:
Well since `#analysis` is only a child view so I had to indent everything. But you're right, for some reasons tabs were used in some parts, despite setting Sublime to indent using spaces...